src/impl/pooling_direct.jl

```diff
@@ -178,10 +192,15 @@ for name in (:max, :mean)
 
         # If we're doing mean pooling, we represent division by kernel size by rolling
         # it into the `alpha` multiplier.
-        if $(name == :mean)
+        if $(name == :mean) || $(name == :lp)
             alpha = alpha / prod(K)
```

Comment:
If the output is supposed to be `y = (∑ x^p)^(1/p)`, we shouldn't divide by prod(K), right?

Reply:
Oh yes, I forgot to delete this. Thank you!